rune/libenclave/intelsgx/arch.go

```diff
@@ -128,26 +129,29 @@ type Report struct {
 }
 
 type ReportBody struct {
-	CpuSvn       [16]byte `struct:"[16]byte"`
-	MiscSelect   uint32   `struct:"uint32"`
-	_            [12]byte `struct:"[12]byte"`
+	CpuSvn     [16]byte `struct:"[16]byte"`
+	MiscSelect uint32   `struct:"uint32"`
+	_          [12]byte `struct:"[12]byte"`
+	// IsvExtProdId is reserved in Quote V3
```

Comment:
The quote spec for ecdsa has a wrong format definition, plz ignore it. Just remove this comment line and keep the previous code.